### DISCUSSION TITLE
¿Alguien ya ingreso satisfactoriamente a la Organización de "LAUNCH X LATAM" en Github?

### DISCUSSION THREAD

**Discussion by MauZubi** [Launch-X-Latam/OnBoarding#7] [Q&A]
Inivtación ala organización: https://launchxauth.azurewebsites.net/joingh

Ya ingrese mi correo  registrado de mi cuenta en Github para que me llegará la invitación a la organización en Github, pero no me llega nada al correo. No se, si haya otra opción por medio del username de Github para solicitar la invitacióin a la organización  "LAUNCH X LATAM"

**Comment by usslx05275** [comment]
Hola, yo también ya lo hice, pero la misma página dice esto:
![image](https://user-images.githubusercontent.com/114427597/192356507-2651302c-4f4f-447a-b89d-e9f1e6b703ac.png)

Así que te aconsejo que hagas  lo del OnBoarding que es esta parte: 
![image](https://user-images.githubusercontent.com/114427597/192356703-babe495c-5ce8-4aa4-a7c9-6f0f8810eb37.png)
 
Y una vez que llenes ese formulario con tus datos, recibirás en tu correo la invitación con un QR y un link, y te permitirá entran en la misión FrontEnd desde teams. 

Espero te funcione :)

**Reply by ghost** [reply]
a mi me pasa lo mismo u.u .. pero no me deja entrar al grupo de la Mision en Teams

**Reply by EduGarcia502** [reply]
hola que tal explorers, ya me loguie correctamente

**Reply by Gl096** [reply]
si team, cuando hayan completado el form ingresen con su cuenta de innovacion.mx
 y les saldrá el QR :D

**Reply by J0SE-ESC0BAR** [reply]
Que repositorio colocan en el formulario

**Reply by TlacaelelMendez** [reply]
En este momento el link funciona, recuerden ingresar el **correo con el que crearon su cuenta de Git**, en mi caso fue el de mi universidad no el de inovaccion porque ya tenia cuenta creada desde tiempo atras. Espero les sirva, saludos!

**Reply by DannyPalma22** [reply]
Hice todos los pasos , pero no me aceptan el Teams, alguien me podría ayudar por favor

**Reply by J0SE-ESC0BAR** [reply]
@DannyPalma22  x2, a mi tampoco me aparece, ya les escribi al instagram pero aun no me han contestado

**Comment by gumodi35** [comment]
Si ya completaron deberían recibir en automático, recuerden usar la misma cuenta con la que se registraron para que no les cause conflicto.
